app/src/main/java/com/p2p/framework/InstructionsLocalResourcesSource.kt

```diff
@@ -0,0 +1,22 @@
+package com.p2p.framework
+
+import android.content.Context
+import com.p2p.data.instructions.InstructionsSource
+import com.p2p.presentation.extensions.getString
+import com.p2p.presentation.home.games.Game
+
+/**Instructions for all games*/
+class InstructionsLocalResourcesSource(private val context: Context) : InstructionsSource {
+
+    private val instructionsByGame = mapOf(
+        Game.TUTTI_FRUTTI to getForGame(Game.TUTTI_FRUTTI.instructionsRes)
+    )
+
+    private fun getForGame(resourceId: Int): String =
+        context.resources.openRawResource(resourceId).getString()
+
+    override fun getInstructions(game: Game): String =
+        instructionsByGame[game]!!
```

Comment:
Yo lo simplificaría a:

```suggestion
    override fun getInstructions(game: Game): String = getForGame(game)
```